### ISSUE TITLE
VMAF score of VP9 video is less than H.264 at 500Kbps using highest quality settings on each

### ISSUE THREAD

**Issue by singhkays** [Netflix/vmaf#323]
I cut a sequence from 00:00:30 to 00:00:34 from Tears of Steel 1080p using ffmpeg

Here are the ffmpeg 2-pass settings for 

**x264 - VMAF score 72.8**
```
./ffmpeg -i scene1.mp4 -c:v libx264 -an -b:v 500k -filter:v scale=720:-1 -preset placebo -pass 1 -f mp4 -y /dev/null && ./ffmpeg -i scene1.mp4 -movflags +faststart -pix_fmt yuv420p -c:v libx264 -an -b:v 500k -filter:v scale=720:-1 -pass 2 -preset placebo ./x264/scene1/x264_500k.mp4
```

**vp9 - VMAF score 67.1**

```
./ffmpeg -i scene1.mp4 -c:v libvpx-vp9 -an -b:v 500k -filter:v scale=720:-1 -row-mt 1 -quality best -cpu-used 0 -tile-columns 2 -threads 8 -pass 1 -f webm -y /dev/null && ./ffmpeg -i scene1.mp4 -pix_fmt yuv420p -c:v libvpx-vp9 -an -b:v 500k -filter:v scale=720:-1 -row-mt 1 -quality best -cpu-used 0 -tile-columns 2 -threads 8 -pass 2 ./vp9/scene1/vp9_500k.webm
```

This seems completely unexpected! I've attached both encoded files below

[encoded-files.zip](https://github.com/Netflix/vmaf/files/3075485/encoded-files.zip)

VMAF was then measure with the following command
```
../../ffmpeg -i vp9_500k.webm -i ../../scene1.mp4 -filter_complex "[0:v]scale=1920x800:flags=bicubic[main];[main][1:v]libvmaf=model_path=/home/kay/vmaf/model/vmaf_v0.6.1.pkl" -f null - >>vmaf-scene1.txt -hide_banner
```

**Comment by li-zhi** [comment]
Hi I looked into the two encoded videos, and to be honest, the perceptual quality of the two are rather indistinguishable. (The x264 encode is probably slightly inferior when watching frame-to-frame).

Then the question is how much the VMAF difference of 5.7 can tell. I would recommend running VMAF again with confidence interval enabled. In brief, you should compare the difference with 1.96 * BOOTSTRAP_VMAF_stddev_score. If the difference is smaller, it means that with high probability (95%), one cannot draw the conclusion that the quality of the two videos are different. See more here:
https://github.com/Netflix/vmaf/blob/master/resource/doc/conf_interval.md

**Comment by singhkays** [comment]
@li-zhi Thank you for taking a look. I figured out the issue. My ffmpeg encodes were causing ffmpeg to throw out an error of type `#invalid length 0x11 > 0x3dfb4 in parent` but the file was getting encoded. Someone on Doom9 forum mentioned frames weren't aligned in the encodes https://forum.doom9.org/showthread.php?p=1871661#post1871661

So, instead of using a `.webm` container, I used a `.mp4` container and the error went away and I have expected VMAF scores 
`72.8 for x264 vs 74.7 for VP9`
